README.adoc

```diff
@@ -39,6 +39,18 @@ And publish your changes (if you got authorization, otherwise Jenkins will do it
 $ rake clean build publish
 ```
 
+==== Troubleshooting:
+
+* Error message `Could not find ffi-1.13.1 in any of the sources`
++
+If you get that error message, run:
++
+```
+sudo gem pristine ffi --version 1.13.1
```

Comment:
After the awestruct upgrade, it didn't just work for me. I had to run this command (and the rake setup below it).